Code/GraphMol/Wrap/rdchem.cpp

```diff
@@ -59,6 +144,23 @@ BOOST_PYTHON_MODULE(rdchem) {
       &translate_value_error);
   python::register_exception_translator<RDKit::MolSanitizeException>(
       &rdSanitExceptionTranslator);
+
+  python::def("WrapLogs", WrapLogs,
+              "Wrap the internal RDKit streams so they go to python's "
+              "SysStdErr");
+  python::def("LogWarningMsg", RDLogWarning,
+              "Log a warning message to the RDKit warning logs");
+  python::def("LogInfoMsg", RDLogInfo,
+              "Log a warning message to the RDKit info logs");
+  python::def("LogErrorMsg", RDLogError,
+              "Log a warning message to the RDKit error logs");
+  python::def("LogDebugMsg", RDLogDebug,
+              "Log a warning message to the RDKit debug logs");
+
+#ifdef RDK_TEST_MULTITHREADED
+  python::def("LogThreadTest", LogThreadTest);
```

Comment:
I needed to test this somehow, I'll try to find a better spot.  Perhaps a throwaway python module?